### ISSUE TITLE
No module named 'en'

### ISSUE THREAD

**Issue by AliRehanCodes** [gunthercox/ChatterBot#2140]
**I'm using chatterbot to build a ChatBot, but I met this error when I run my code and I don't know how to deal with it, please help me . my code is:**

from chatterbot import ChatBot
from chatterbot.trainers import ListTrainer

bot = ChatBot('My Bot')

convo = [
    'hello',
    'hi there !',
    'what is your name ?',
    'My name is Bot , i am created by Durgesh',
    'how are you ?',
    'I am doing great these days',
    'thank you',
    'In which city you live ?',
    'I live in lucknow',
    'In which language you talk?',
    ' I mostly talk in english'
]

trainer = ListTrainer(bot)
trainer.train(convo)

answer = bot.get_response("What is your name?")
print(answer)

**Comment by matheusfillipe** [comment]
Take a look at https://github.com/gunthercox/ChatterBot/issues/2139#issuecomment-813152490 and https://stackoverflow.com/a/66087946/5731541

**Comment by codePerfectPlus** [comment]
```bash
sudo python3 -m pip install spacy download en_core_web_sm
sudo python3 -m pip install spacy download en
```

It will work

**Comment by MyBeets** [comment]
for me it still says ModuleNotFoundError: No module named 'en'
after those two downloads

**Comment by codePerfectPlus** [comment]
@MyBeets Try with `Sudo` command,

If you are on windows os, open cmd as `administrator` then run commands.

**Comment by MyBeets** [comment]
yeah no worries I got it working by deleting spacy and reinstalling it

**Comment by gunthercox** [comment]
I believe this error should be fully resolved once the changes from #2389 are merged in.
